### PR TITLE
add mosquito module.

### PR DESCRIPTION
**Pull request by mosquito-michaelangelo**
added my module in main.py

### CODE REVIEW

File: errawrs/mosquito.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        self.value_2 = 0
+        self.total = 0
+
+    def get_number(self, ):
+        self.value_1 = float(input("Enter a number: "))
+        self.value_2 = float(input("Enter a number: "))
+
+        print("\n")
+
+    def sum(self):
```
Comment (victorioso-daniel-2) [inline errawrs/mosquito.py:13]: verb form for methods

**Review by victorioso-daniel — CHANGES_REQUESTED** [review]
use error handling for the ff:
when user input an invalid character
when user chose division without inputing numbers

File: errawrs/mosquito.py (hunk 1 above)
Comment (mosquito-michaelangelo) [reply errawrs/mosquito.py]: change to verb form

**Comment by mosquito-michaelangelo** [comment]
> [ ](/mosquito-michaelangelo)

use error handling for inputs and division

**Review by victorioso-daniel — APPROVED** [review]
LGTM

**Pull request merged**